packages/components/src/components/Select/Select.stories.tsx

```diff
@@ -0,0 +1,186 @@
+import type { ComponentMeta } from "@storybook/react";
+import React from "react";
+import { useFormik } from "formik";
+import { useForm, SubmitHandler, Controller } from "react-hook-form";
+import { useUID } from "react-uid";
+import { HelpText } from "../HelpText";
+import { Label } from "../Label";
+import { Box } from "../../primitives/Box";
+import { Button } from "../Button";
+import { Select } from "./Select";
+import type { SelectProps } from "./Select";
+
+export default {
+  component: Select,
+  title: "Components/Select",
+} as ComponentMeta<typeof Select>;
+
+const selectItems: SelectProps["items"] = [
+  { value: "Select an option", initial: true },
+  { value: "Option One" },
+  { value: "Option Two" },
+  { value: "Option Three", disabled: true },
+  { value: "Option Four" },
+  {
+    value:
+      "Option Five which is a really long item and should overflow to the next line",
+  },
+];
+
+export const Default = (): JSX.Element => {
+  const selectID = useUID();
+  const helpTextID = useUID();
+  return (
+    <Box.form>
+      <Label htmlFor={selectID}>Choose One</Label>
+      <Select id={selectID} items={selectItems} name="select" />
+      <HelpText id={helpTextID}>Please choose one of the values.</HelpText>
+    </Box.form>
+  );
+};
+
+export const Large = (): JSX.Element => {
+  const selectID = useUID();
+  const helpTextID = useUID();
+  return (
+    <Box.form>
+      <Label htmlFor={selectID}>Choose One</Label>
+      <Select id={selectID} items={selectItems} name="select" size="large" />
+      <HelpText id={helpTextID}>Please choose one of the values.</HelpText>
+    </Box.form>
+  );
+};
+
+export const Required = (): JSX.Element => {
+  const selectID = useUID();
+  const helpTextID = useUID();
+  return (
+    <Box.form>
+      <Label htmlFor={selectID} required>
+        Choose One
+      </Label>
+      <Select id={selectID} items={selectItems} name="select" required />
+      <HelpText id={helpTextID}>Please choose one of the values.</HelpText>
+    </Box.form>
+  );
+};
+
+export const Disabled = (): JSX.Element => {
+  const selectID = useUID();
+  const helpTextID = useUID();
+  return (
+    <Box.form>
+      <Label htmlFor={selectID}>Choose One</Label>
+      <Select disabled id={selectID} items={selectItems} name="select" />
+      <HelpText id={helpTextID}>Please choose one of the values.</HelpText>
+    </Box.form>
+  );
+};
+
+export const Error = (): JSX.Element => {
+  const selectID = useUID();
+  const helpTextID = useUID();
+  return (
+    <Box.form>
+      <Label htmlFor={selectID}>Choose One</Label>
+      <Select hasError id={selectID} items={selectItems} name="select" />
+      <HelpText hasError id={helpTextID}>
+        This is an error message.
+      </HelpText>
+    </Box.form>
+  );
+};
+
+export const LongItem = (): JSX.Element => {
```

Comment:
I think it's good having a story to test the mobile version of this one. What do you think?

Reply:
Yeah. Worth having that screenshot there. Its been added.